worldedit-core/src/main/java/com/fastasyncworldedit/core/util/EditSessionBuilder.java

```diff
@@ -1,78 +1,18 @@
 package com.fastasyncworldedit.core.util;
 
-import com.fastasyncworldedit.core.Fawe;
-import com.fastasyncworldedit.core.FaweCache;
-import com.fastasyncworldedit.core.configuration.Caption;
-import com.fastasyncworldedit.core.configuration.Settings;
-import com.fastasyncworldedit.core.extent.FaweRegionExtent;
-import com.fastasyncworldedit.core.extent.HistoryExtent;
-import com.fastasyncworldedit.core.extent.MultiRegionExtent;
-import com.fastasyncworldedit.core.extent.NullExtent;
-import com.fastasyncworldedit.core.extent.SingleRegionExtent;
-import com.fastasyncworldedit.core.extent.SlowExtent;
-import com.fastasyncworldedit.core.extent.StripNBTExtent;
-import com.fastasyncworldedit.core.extent.processor.HeightmapProcessor;
-import com.fastasyncworldedit.core.extent.processor.LimitExtent;
-import com.fastasyncworldedit.core.extent.processor.lighting.NullRelighter;
-import com.fastasyncworldedit.core.extent.processor.lighting.RelightMode;
-import com.fastasyncworldedit.core.extent.processor.lighting.RelightProcessor;
-import com.fastasyncworldedit.core.extent.processor.lighting.Relighter;
-import com.fastasyncworldedit.core.history.DiskStorageHistory;
-import com.fastasyncworldedit.core.history.MemoryOptimizedHistory;
-import com.fastasyncworldedit.core.history.RollbackOptimizedHistory;
-import com.fastasyncworldedit.core.history.changeset.AbstractChangeSet;
-import com.fastasyncworldedit.core.history.changeset.BlockBagChangeSet;
-import com.fastasyncworldedit.core.history.changeset.NullChangeSet;
-import com.fastasyncworldedit.core.object.FaweLimit;
-import com.fastasyncworldedit.core.queue.IQueueChunk;
-import com.fastasyncworldedit.core.queue.IQueueExtent;
-import com.fastasyncworldedit.core.queue.implementation.ParallelQueueExtent;
-import com.fastasyncworldedit.core.regions.RegionWrapper;
-import com.fastasyncworldedit.core.wrappers.WorldWrapper;
-import com.sk89q.worldedit.EditSession;
-import com.sk89q.worldedit.WorldEdit;
-import com.sk89q.worldedit.entity.Player;
-import com.sk89q.worldedit.event.extent.EditSessionEvent;
-import com.sk89q.worldedit.extension.platform.Capability;
-import com.sk89q.worldedit.extent.Extent;
-import com.sk89q.worldedit.extent.inventory.BlockBag;
 import com.sk89q.worldedit.internal.util.LogManagerCompat;
-import com.sk89q.worldedit.regions.Region;
-import com.sk89q.worldedit.util.Identifiable;
-import com.sk89q.worldedit.util.eventbus.EventBus;
-import com.sk89q.worldedit.util.formatting.text.TextComponent;
 import com.sk89q.worldedit.world.World;
 import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.Nullable;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import java.util.Locale;
-import java.util.UUID;
-
-import static com.google.common.base.Preconditions.checkNotNull;
 
+//TODO Migrate all logic to com.sk89q.worldedit.EditSessionBuilder
```

Comment:
See my previous comment on the FaweAPI class how this could be handled without an API breakage.